package.json

```
@@ -9,7 +9,7 @@
   "main": "index.js",
   "scripts": {
     "test": "make test",
-    "dynamodb-local": "./test/bin/dynamodb-local"
+    "localstack": "docker run -it -p 4569:4569 -e SERVICES=dynamodb --rm localstack/localstack:0.11.5"
```

Should this just be 
```suggestion
    "localstack": "docker run -it -p 4569:4569 -e SERVICES=dynamodb --rm localstack/localstack"
```

since that's what you have running in travis?

---

Good catch! This is why the build was failing. `localstack` changed ports after `0.11.5`. I'm pinning for now, but we should probably update once a tagged version lands.